backend/compact-connect/tests/app/test_pipeline.py

```diff
@@ -58,29 +58,58 @@ def test_synth_pipeline(self):
         # We must confirm that these scopes are being explicitly created for each compact
         # which are absolutely critical for the system to function as expected.
-        self.assertEqual(self.context['compacts'], list(resource_servers.keys()))
+        self.assertEqual(sorted(self.context['compacts']), sorted(list(resource_servers.keys())))
 
         for compact, resource_server in resource_servers.items():
-            # for this test, we just get the 'aslp' compact resource server
             resource_server_properties = self.get_resource_properties_by_logical_id(
                 persistent_stack.get_logical_id(resource_server.node.default_child),
                 persistent_stack_template.find_resources(CfnUserPoolResourceServer.CFN_RESOURCE_TYPE_NAME),
             )
-            # Ensure the resource servers are created with the expected scopes
+            # Ensure the compact resource servers are created with the expected scopes
             self.assertEqual(
                 ['admin', 'write', 'readGeneral', 'readSSN'],
                 [scope['ScopeName'] for scope in resource_server_properties['Scopes']],
                 msg=f'Expected scopes for compact {compact} not found',
             )
 
+    def test_synth_generates_jurisdiction_resource_servers_with_expected_scopes_for_staff_users(self):
+        persistent_stack = self.app.pipeline_stack.test_stage.persistent_stack
+        persistent_stack_template = Template.from_stack(persistent_stack)
+
+        # Get the jurisdiction resource servers created in the persistent stack
+        resource_servers = persistent_stack.staff_users.jurisdiction_resource_servers
+        # We must confirm that these scopes are being explicitly created for each active jurisdiction
+        # which are absolutely critical for the system to function as expected.
+        # If a new jurisdiction is made active within the system, this test will need to be updated
+        jurisdiction_resource_server_config = []
+        for _jurisdiction, resource_server in resource_servers.items():
+            resource_server_properties = self.get_resource_properties_by_logical_id(
+                persistent_stack.get_logical_id(resource_server.node.default_child),
+                persistent_stack_template.find_resources(CfnUserPoolResourceServer.CFN_RESOURCE_TYPE_NAME),
+            )
+            # remove dynamic user pool id
+            del resource_server_properties['UserPoolId']
+            jurisdiction_resource_server_config.append(resource_server_properties)
+
+        # sort the resource server list by jurisdiction for consistency
+        jurisdiction_resource_server_config.sort(key=lambda jurisdiction: jurisdiction['Identifier'])
+        # sort the scopes within the resource server by name for consistency
+        for resource_server in jurisdiction_resource_server_config:
+            resource_server['Scopes'].sort(key=lambda scope: scope['ScopeName'])
+        self.compare_snapshot(
```

Comment:
I think an explanation here would be helpful. Am I right that we have no `coun.*` scopes in this snapshot because, while the compact is active in the `test` environment, none of its jurisdictions are so the `ky/coun.*`, etc, scopes are, rightly, being skipped?